examples/get_iss_data.py

```diff
@@ -16,97 +19,121 @@ def download(input_dir, url):
     z.extractall(input_dir)
 
 
-def load_tiff_stack(path):
-    stack = imread(path)
-    return stack
+def build_hybridization_stack(input_dir, output_dir):
+    prefix = "fov_0"
 
+    imgs = list()
+    for hyb in range(1, 5):
+        hyb_round = list()
+        for ch in range(2, 6):
+            hyb_round.append(imread(os.path.join(input_dir, str(hyb), "c{}.TIF".format(ch))))
```

Comment:
I would find a way to do reading and writing inside the for loop that starts below on line 37 so that you only have one image in memory at a time. This could run into problems if we mirror it for larger datasets. 

If you store the input file stem in `imgs` instead of the loaded image, I think that would suffice.